openmetadata-ui/src/main/resources/ui/cypress/constants/constants.ts

```diff
@@ -841,3 +978,19 @@ export const ID = {
     api: '/api/v1/services/storageServices?*',
   },
 };
+
+export const ENTITY_PATH = {
+  tables: 'table',
+  topics: 'topic',
+  dashboards: 'dashboard',
+  pipelines: 'pipeline',
+  mlmodels: 'mlmodel',
+  containers: 'container',
+  tags: 'tag',
+  glossaries: 'glossary',
+  searchIndexes: 'searchIndex',
+  storedProcedures: 'storedProcedure',
+  glossaryTerm: 'glossaryTerm',
+  databases: 'database',
+  databaseSchemas: 'databaseSchema',
+};
```

Comment:
Can you check if we can avoid this const?
Avoiding too many entityType related constant to avoid confusion on which one to use

Reply:
As discussed, it is not feasible to use existing const and enums, so we have to stick with `ENTITY_PATH`.